plugins/newsdownloader.koplugin/feeds.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Yes, I had thought about that. But since:
1. **Lua files are specyfic for lua** and **XML files are more general/standard and used all over Internet and PC**
2. you don't have to be famirial with LUA to edit XML files
3. so more people are able to edit XML over LUA files
4. you don't have to be a lua developer to edit XML files
5. newsConfig contains other kind of configuration (folder and filenames used by program).
6. on Android we wouldn't be able to edit newsConfig.lua (**this is one of the most important for me as I'm running android)** so we wouldn't be able to change default feeds (in current implementation), [on Android feed xml config is copied to writeable location) 
7. I don't know howto create a clean and meaningful list in LUA config file & don't have time to learn more LUA

I'd like to leave it as it is. Is that ok?

It would make sense to change it to configuration is we had GUI for editing feed urls, but I currently don't plan to do so, maybe in some time in future if I had time.

So please for first edition approve it this way, if I had time in future I'll try to change it to UI edition. But it might be in couple months, year or more. Let's not wait so long for this pull request. Ok?